### DISCUSSION TITLE
Welcome to ai.assistant Discussions!

### DISCUSSION THREAD

**Discussion by josoavj** [josoavj/aichat#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

**Comment by sudoted** [comment]
**Hello there**
